Source/Project64-core/N64System/N64DiskClass.cpp

```diff
@@ -290,7 +290,7 @@ void CN64Disk::ConvertDiskFormat()
         atrack = 0;
         for (track = 0; track < ZoneTracks[zone]; track++)
         {
-            if (track == SystemData[0x20 + zone * 0xC + atrack])
+            if (atrack < 0xC && track == SystemData[0x20 + zone * 0xC + atrack])
```

Comment:
Out of curiosity, what does '0xC' imply?
Would it be worth to create a constant to name the value meaningfuly?